tests/lib/rules/newline-after-var.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @fileoverview Tests for newline-after-var rule.
+ * @author Gopal Venkatesan <http://g13n.me>
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var eslint = require("../../../lib/eslint"),
+    ESLintTester = require("eslint-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var eslintTester = new ESLintTester(eslint);
+
+eslintTester.addRuleTest("lib/rules/newline-after-var", {
+    valid: [
+        { code: "var greet = 'hello'; console.log(greet);", args: [2, "never"] },
```

Comment:
@nzakas the `[2: "never"]` is for disallowing newlines and not for disabling this feature.